AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/CallActivity.java

```diff
@@ -233,7 +233,9 @@ private void initParticipantViews() {
         localParticipantView = new ParticipantView(this);
         localParticipantView.setDisplayName(callingContext.getDisplayName() + " (Me)");
         localParticipantView.setVideoDisplayed(callingContext.getCameraOn());
+        localParticipantView.setSwitchCameraButtonDisplayed(callingContext.getCameraOn());
         localParticipantView.setIsMuted(!callingContext.getMicOn());
+        localParticipantView.setImageButtonOnClickAction(this::switchCameraAsync);
```

Comment:
do we need to unsubscribe on Destroy?

Reply:
> do we need to unsubscribe on Destroy?

`localParticipantView` is only created once, and so setting the OnClickAction is also called once.  Additionally, this just sets the Runnable, so this particular method running multiple times is idempotent.